pkgs/development/mobile/androidenv/generated/system-images-google_apis_playstore.nix

```diff
@@ -47,18 +47,116 @@
+    };
+  };
   "28".google_apis_playstore."x86_64" = {
     name = "system-image-28-google_apis_playstore-x86_64";
     path = "system-images/android-28/google_apis_playstore/x86_64";
     revision = "28-google_apis_playstore-x86_64";
     displayName = "Google Play Intel x86 Atom_64 System Image";
     archives.all = fetchurl {
-      url = "https://dl.google.com/android/repository/sys-img/google_apis_playstore/x86_64-28_r08.zip";
+      url = https://dl.google.com/android/repository/sys-img/google_apis_playstore/x86_64-28_r08.zip;
       sha1 = "a767da996fdea7a1f5632a9206fa5c009d6e580c";
     };
   };
-}
+  "29".google_apis_playstore."x86" = {
+    name = "system-image-29-google_apis_playstore-x86";
+    path = "system-images/android-29/google_apis_playstore/x86";
+    revision = "29-google_apis_playstore-x86";
+    displayName = "Google Play Intel x86 Atom System Image";
+    archives.all = fetchurl {
+      url = https://dl.google.com/android/repository/sys-img/google_apis_playstore/x86-29_r08-windows.zip;
+      sha1 = "1c45e690e9ee6a44f40549e9fb68d3fd52ba4970";
+    };
+  };
+  "29".google_apis_playstore."x86" = {
+    name = "system-image-29-google_apis_playstore-x86";
+    path = "system-images/android-29/google_apis_playstore/x86";
+    revision = "29-google_apis_playstore-x86";
+    displayName = "Google Play Intel x86 Atom System Image";
+    archives.all = fetchurl {
+      url = https://dl.google.com/android/repository/sys-img/google_apis_playstore/x86-29_r08-darwin.zip;
+      sha1 = "1c45e690e9ee6a44f40549e9fb68d3fd52ba4970";
+      url = https://dl.google.com/android/repository/sys-img/google_apis_playstore/x86-29_r08-linux.zip;
+      sha1 = "1c45e690e9ee6a44f40549e9fb68d3fd52ba4970";
+    };
+  };
+  "29".google_apis_playstore."x86_64" = {
+    name = "system-image-29-google_apis_playstore-x86_64";
+    path = "system-images/android-29/google_apis_playstore/x86_64";
+    revision = "29-google_apis_playstore-x86_64";
+    displayName = "Google Play Intel x86 Atom_64 System Image";
+    archives.all = fetchurl {
+      url = https://dl.google.com/android/repository/sys-img/google_apis_playstore/x86_64-29_r08-windows.zip;
+      sha1 = "94835980b4a6eaeeb41936d7fb1381698e48433a";
+    };
+  };
+  "29".google_apis_playstore."x86_64" = {
+    name = "system-image-29-google_apis_playstore-x86_64";
+    path = "system-images/android-29/google_apis_playstore/x86_64";
+    revision = "29-google_apis_playstore-x86_64";
+    displayName = "Google Play Intel x86 Atom_64 System Image";
+    archives.all = fetchurl {
+      url = https://dl.google.com/android/repository/sys-img/google_apis_playstore/x86_64-29_r08-darwin.zip;
+      sha1 = "94835980b4a6eaeeb41936d7fb1381698e48433a";
+      url = https://dl.google.com/android/repository/sys-img/google_apis_playstore/x86_64-29_r08-linux.zip;
+      sha1 = "94835980b4a6eaeeb41936d7fb1381698e48433a";
+    };
+  };
+  "R".google_apis_playstore."x86" = {
```

Comment:
Dup of the above, for the `*-windows.zip` 

The one below has `*-darwin.zip` and `*-linux.zip`

Maybe these were meant to be combined into one set, and `archives.all` should be `archives = { windows= fetchurl{..}; darwin = fetchurl {...}; linux = fetchurl {...}; }`